flow/project.py

```diff
@@ -1042,7 +1042,8 @@ def _print_progress(x):
 
     @_support_legacy_api
     def print_status(self, jobs=None, overview=True, overview_max_lines=None,
-                     detailed=False, parameters=None, skip_active=False, param_max_width=None,
+                     detailed=False, parameters=None, all_parameters=False,
```

Comment:
Instead of adding an additional argument here, I'd suggest to use a special constant as argument to `parameters`. In this way we avoid breaking backwards-compatibility and the ambiguity when users provide both the `all_parameters` and the `parameters `argument.

So my suggestion is to add the following line somewhere at the top of the module:
```
PRINT_STATUS_ALL_PARAMETERS = True
```
And then check for that in line 1173: `if parameters is PRINT_STATUS_ALL_PARAMETERS:`.